server/app/services/geo/esri/EsriClient.java

```diff
@@ -330,7 +349,7 @@ public CompletionStage<ImmutableList<ServiceAreaInclusion>> getServiceAreaInclus
               List<String> features =
                   ctx.read(
                       "features[*].attributes." + esriServiceAreaValidationOption.getAttribute());
-
+              System.out.println("features = " + features);
```

Comment:
Was this for debugging? Probably should remove it and use the logger if we need to keep the message.